packages/rum/test/benchmarks/profiler.js

```diff
@@ -51,14 +51,9 @@ function gatherRawMetrics(browser, url) {
      */
     let client = null
     try {
-      client = await browser.pageTarget(page).createCDPSession()
+      client = await context.newCDPSession(page)
     } catch (_) {}
 
-    /**
-     * Disable cache for each run
-     */
-    await page.setCacheEnabled(false)
```

Comment:
Is the cache disabled by default in the new version?

Reply:
Yes see this https://github.com/microsoft/playwright/pull/1231